apps/rd/rd-judicial-api/ithc.yaml

```diff
@@ -55,4 +55,4 @@ spec:
         POSTGRES_USERNAME: pgadmin
         POSTGRES_NAME: dbjuddata
         POSTGRES_PORT: 5432
-        DB_READER_USERNAME: pgadmin
+        DB_READER_USERNAME: "pgadmin"
```

Comment:
If this is the DB reader rather than the admin user, I would expect this to be `DTS CFT DB Access Reader` for ITHC and other non-Prod envs.